models/stg_reddit_ads__account_report.sql

```diff
@@ -30,11 +29,11 @@ final as (
         conversion_roas,
         cpc,
         ctr,
-        date_day, -- renamed in macro
+        date_day, -- renamed in macro 
```

Comment:
What was the main reason this needed to be renamed in the macro? I think we have been able to rename it without error in the other ad packages.

Regardless, if we are renaming in the macro then we should provide the exact path in the comment to where this renaming has taken place.

Reply:
There actually wasn't a specific reason, I was working on a package previously that involved renaming in the macro so I think that just carried over haha. I actually updated to renaming in the staging model, since I think that's clearer for the end user